_unittest/test_02_3D_modeler.py

```diff
@@ -1052,7 +1052,7 @@ def test_59a_region_property(self):
 
     def test_59b_region_property_failing(self):
         self.aedtapp.modeler.create_air_region()
-        assert not self.aedtapp.modeler.change_region_coordinate_system(region_cs="NoCS")
+        assert not self.aedtapp.modeler.change_region_coordinate_system(coordinate_system="NoCS")
```

Comment:
why not using simply "name"? the fact that is a coordinate system is already in method name

Reply:
if you look at the implemantation I already used "name" for the region name:
`@pyaedt_function_handler(region_cs="coordinate_system", region_name="name")`
`def change_region_coordinate_system(self, coordinate_system="Global", name="Region")`
@maxcapodi78

Reply:
in such case better to use assignment.